taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+DRIVER_LICENSE_VALIDATOR = RegexValidator(
+    regex=r"^[A-Z]{3}[\d]{5}$",
+    message="Enter a valid value license number:"
+            " 1.Consist only of 8 characters"
+            " 2.First 3 characters are uppercase letters"
+            " 3.Last 5 characters are digits", )
+
+
+class DriverCreationForm(UserCreationForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[DRIVER_LICENSE_VALIDATOR, ]
+    )
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name",
+            "last_name",
+            "license_number",
+        )
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[DRIVER_LICENSE_VALIDATOR, ]
+    )
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The queryset should be filtered to include only `Driver` instances, not all user models. Consider using `Driver.objects.all()` instead of `get_user_model().objects.all()`.